apollo-federation/src/query_graph/mod.rs

```diff
@@ -134,18 +134,24 @@ impl TryFrom<QueryGraphNodeType> for ObjectTypeDefinitionPosition {
     }
 }
 
-/// Contains all of the data necessary to connect the object field (`coordinate`) with the
-/// `@fromContext` to its (grand)parent types contain a matching selection.
+/// Contains all of the data necessary to connect the object field argument (`argument_coordinate`)
+/// with the `@fromContext` to its (grand)parent types contain a matching selection.
 #[derive(Debug, PartialEq, Clone)]
 pub struct ContextCondition {
     context: String,
     subgraph_name: Arc<str>,
+    // This is purposely left unparsed in query graphs, due to @fromContext selection sets being
+    // duck-typed.
     selection: String,
     types_with_context_set: IndexSet<CompositeTypeDefinitionPosition>,
+    // PORT_NOTE: This field was renamed because the JS name (`namedParameter`) left confusion to
+    // how it was different from the argument name.
+    argument_name: Name,
     // PORT_NOTE: This field was renamed because the JS name (`coordinate`) was too vague.
     argument_coordinate: ObjectFieldArgumentDefinitionPosition,
-    named_parameter: Name,
```

Comment:
Refactors:
- Some renames here for clarity (argument instead of parameter to make it clear it's an actual GraphQL argument).